src/languageservice/serviceclient.ts

```diff
@@ -288,9 +291,9 @@ export default class SqlToolsServiceClient {
         client.onReady().then( () => {
             this.checkServiceCompatibility();
 
+            client.onNotification(LanguageServiceContracts.TelemetryNotification.type, this.handleLanguageServiceTelemetryNotification());
```

Comment:
This might risk missing some early telemetry notification, but I don't think that's a major risk.

Reply:
I think we've turned-off telemetry for the extension at this point, so no risk for the current release.  When moving to the v5 language client I'll look into refactoring how we register notification handlers.